DAS/das/utils/mqtt_test.py

```diff
@@ -0,0 +1,116 @@
+import sys
```

Comment:
What makes this a `util` and `mqtt_playback`/`fake_module` not, despite them being effectively the same thing, just for different bike versions? I feel like this could be in the directory above.

Also while we're moving it, we should rename it too. I'm tempted to suggest `mqtt_playback_v2.py`... Although that could clearly lead to some confusion due to the original name of the second iteration of your logger. Open to suggestions, would be pretty easy to beat `mqtt_test.py`

Reply:
I think that is a very good idea actually, I didn't realise that `mqtt_test` was essentially a mqtt playback tool. It definitely should not be in the `util` folder.

I think the easiest would be to name all of the tools that I wrote as "V3" and rename `mqtt_test.py` to `mqtt_playback_v2.py`

Reply:
After some thought I have decided to prepend all files corresponding to a version of the logger/recorder with either "V3" or "V2". This also makes it really easy to search tools now.